### ISSUE TITLE
[Python APIView] Add docstrings to tokenfile

### ISSUE THREAD

**Issue by tjprescott** [Azure/azure-sdk-tools#2756]
Select C# APIViews already support a "show documentation" checkbox. Thus, presumably if we supplied that information for Python, it would "just work".

**Comment by JonathanGiles** [comment]
This is correct. All Java libraries include the JavaDoc too - happy to give pointers on how we do this if needed.
